commands/cms/lighthouseScore.ts

```diff
@@ -32,7 +31,7 @@ const DEFAULT_TABLE_HEADER = [
 ];
 
 exports.command = 'lighthouse-score [--theme]';
-exports.describe = false; // i18n(`${i18nKey}.describe`);
+exports.describe = i18n(`${i18nKey}.describe`);
```

Comment:
I think I just un-hid it, because I was using the help menu to aid in testing. I'm going to re-hide it for now and alert the CMS team--it'd be better if they did it in a separate PR.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1212/commits/11c8c119479c484027f71dffa1ad97d64951d2d8